extensions/phare-io-uptime/src/types.ts

```diff
@@ -0,0 +1,52 @@
+export interface Assertion {
+  type: string;
+  operator: string;
+  value: string | number;
+}
+
+export interface Monitor {
+  id: number;
+  name: string;
+  status: string;
+  paused: boolean;
+  protocol: string;
+  request: {
+    url: string;
+    method: string;
+    tls_skip_verify: boolean;
+    follow_redirects: boolean;
+  };
+  interval: number;
+  timeout: number;
+  success_assertions: Assertion[];
+  incident_confirmations: number;
+  recovery_confirmations: number;
+  regions: string[];
+  response_time: number;
+  updated_at: string;
+  created_at: string;
+}
+
+export interface Preferences {
+  phareApiKey: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`